Class-14-(SQL-Part-1).md

```diff
@@ -0,0 +1,26 @@
+# Class 14: Databases and SQL
```

Comment:
Don't forget to update the home page to add the class 14 link.

Reply:
Done! Thanks!